modules/checkout/src/Plugin/Commerce/CheckoutFlow/CheckoutFlowBase.php

```diff
@@ -166,6 +180,22 @@ public function getVisibleSteps() {
   /**
    * {@inheritdoc}
    */
+  public function buildCartSummary(array $form, FormStateInterface $form_state) {
+    $summary_view_id = $this->configuration['cart_summary_view'];
```

Comment:
No point in declaring the $summary_view_id variable if it's only used once in the returned array?